src/models/foreignIndirectInterest.js

```diff
@@ -0,0 +1,43 @@
+import { foreignCoOwnersModel } from 'validators/foreigncoowner'
+
+const foreignIndirectInterest = {
+  InterestTypes: {
+    presence: true,
+    array: {
+      validator: { presence: true },
+      length: { minimum: 1 },
+    },
+  },
+  InterestType: { presence: true, hasValue: true },
+  Firstname: { presence: true, hasValue: true },
+  Lastname: { presence: true, hasValue: true },
+  Relationship: { presence: true, hasValue: true },
+  Acquired: { presence: true, date: { requireDay: false } },
+  HowAcquired: { presence: true, hasValue: true },
+  Cost: { presence: true, hasValue: true },
```

Comment:
previous comment about `numericality` applies to `Cost` and `Value` here as well